src/openforms/js/components/admin/form_design/logic-data.js

```diff
@@ -12,6 +13,7 @@ import {post, put, apiDelete} from '../../../utils/fetch';
  * @return {Array}                     Array of newly created rules
  */
 export const saveRules = async (endpoint, formUrl, csrftoken, rules, rulesToDelete) => {
+    const fieldPrefix = endpoint === LOGICS_ENDPOINT ? 'logicRules' : 'priceRules';
```

Comment:
I don't like how we get specifics back into the generic function here - can't you pass this as an argument?